packages/esm-service-queues-app/src/transition-latest-queue-entry/add-patient-to-queue-modal/add-patient-to-queue-entry.modal.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useCallback, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, Form, ModalBody, ModalFooter, ModalHeader, Stack, InlineNotification } from '@carbon/react';
```

Comment:
Please remove unused imports
```suggestion
import { Button, Form, ModalBody, ModalFooter, ModalHeader } from '@carbon/react';
```